runtime/async/async.go

```diff
@@ -19,6 +21,7 @@ import (
 
 const (
 	defaultPattern = "/"
+	protocolEnvVar = "APP_PROTOCOL"
```

Comment:
if I understand correctly, we will need to pass this environment variable via openfunction controller?

Reply:
Yeah, openfunction controller will set this envVar，when `serving.annotations` contains `dapr.io/app-protocol: http`.